frontend/src/scenes/persons/PersonScene.tsx

```diff
@@ -219,7 +219,7 @@ export function PersonScene(): JSX.Element | null {
                                 query={{
                                     kind: NodeKind.DataTableNode,
                                     full: true,
-                                    hiddenColumns: ['person'],
+                                    hiddenColumns: ['person_display_name -- Person'],
```

Comment:
I was thinking it might be better to make this a constant to avoid similar issues in the future, but wasn't sure where it would live. [constants.tsx](https://github.com/PostHog/posthog/blob/master/frontend/src/lib/constants.tsx) perhaps?

Reply:
Yes, either there or maybe https://github.com/PostHog/posthog/blob/master/frontend/src/queries/utils.ts

Reply:
went for constants based on what was already in utils 👍